### DISCUSSION TITLE
Generate a PDF for publishing

### DISCUSSION THREAD

**Discussion by aburkov** [executablebooks/meta#158] [Q&A]
Hi!

When I learned about this project, I was super excited. I really liked the possibility of "building beautiful, publication-quality books." However, during an investigation, I could not find a lot of detail on how to generate a publication-quality PDF file. All examples I saw looked like a printed HTML page rather than a real book interior that you would create using, for example, LaTeX.

Did I miss something? Please advise.

**Comment by jstac** [comment]
Hi @aburkov , this is very much a part of the goals of the project.  

I'm also involved in writing [these lectures](https://python-programming.quantecon.org/) and maintaining high quality PDFs is always high on our agenda.

Unfortunately, LaTeX is lagging behind html at this point in time, but it's being worked on actively by @mmcky and @AakashGfude.  They can update you on where we are at with LaTeX support.

**Comment by mmcky** [comment]
hi @aburkov the project is at the early stage of building out support for producing `PDF` files via LaTeX. We have connected `jupyter-book` through to the default `sphinx.latexpdf` writer at this stage -- but it is marked as `experimental` and is an active project. We are currently looking into LaTeX theme support for customising PDF output and doing some work to align `html` and `pdflatex` output (i.e. common sections, chapters, parts), in the medium term we are also looking at adding support for `frontmatter` etc.

There is some documentation available here for building `PDF` via `HTML` or `LaTeX`:

https://jupyterbook.org/advanced/pdf.html

Some improved docs are also coming down the pipeline

https://github.com/executablebooks/jupyter-book/pull/1030

**Comment by AakashGfude** [comment]
Hi @aburkov we have created an extension for this very purpose https://github.com/executablebooks/jupyterbook-latex . And are planning to use it as a default for building pdfs with `jupyter-book`. 

Hope it will help you in generating a decent level of latex output. And we will love your inputs and ideas on what other features you would want it to have.
